pkg/import/import_resources.go

```diff
@@ -106,13 +104,12 @@ func Import(rd, definitionsFile io.Reader) ([]string, []string, error) {
 			add = append([]string{arg}, add...)
 			// Append
 			remove = append(remove, a)
-			break
+		} else {
+			// Append
+			add = append(add, arg)
+			// Prepend
+			remove = append([]string{a}, remove...)
 		}
```

Comment:
This also appears left indented compared to the `if` statement line above.

Reply:
Same as above, I'm looking into it

Reply:
Just an impression, could it be because my editor is using <tab 4 spaces> and github is default < 8 spaces> ? May be I have to chose in my editor <use spaces 4> instead of tabs to avoid this mixing.

Reply:
This is an old issue and it's my case https://github.com/microsoft/vscode/issues/62935